core/templates/components/angular-html-bind/angular-html-bind.component.ts

```diff
@@ -0,0 +1,179 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview AngularHtmlBind Component (not associated with reusable
+ * components.)
+ * NB: Reusable component directives should go in the components/ folder.
+ */
+
+// HTML bind component that trusts the value it is given and also evaluates
+// custom component tags in the provided value.
```

Comment:
It no longer does that. We have scoped it pretty well now.

Reply:
comment removed